org.eclipse.lyo.tools.toolchain.design/pom.xml

```diff
@@ -60,7 +60,6 @@
 							<jar>${project.basedir}/../org.eclipse.lyo.tools.toolchain.model/target/org.eclipse.lyo.tools.toolchain.model-${project.parent.version}.jar</jar>
 							<jar>${project.basedir}/../org.eclipse.lyo.tools.vocabulary.model/target/org.eclipse.lyo.tools.vocabulary.model-${project.parent.version}.jar</jar>
 							<jar>${project.basedir}/../org.eclipse.lyo.tools.common/target/org.eclipse.lyo.tools.common-${project.parent.version}.jar</jar>
-                            <jar>p2.eclipse-plugin:org.eclipse.sirius.diagram.model</jar>
```

Comment:
Note: I could not make it bouidl with this jar reference. Removing it  is fine, and it had no effect on the resulting LyoDesigner.